LibertyDSNP/frequency
# Set ProviderBoost capacity generated and fix tests

# Goal
The goal of this PR is to set the amount of capacity generated by a provider boost to the final amount, 50% of what is generated by MaximizedCapacity staking.

Closes #1569 

### Discussion
* Also Fixes some tests broken from the last rebase with main.

### To Verify
Since CI doesn't run for PRs against non-main branches, you can verify by running `make test` and `make start` if you like, and verify that tests all pass and the chain can run locally.

# Checklist
- [x] Tests <del>added</del> updated
